models/cart-discount/src/cart-discount/cart-discount-draft/presets/sample-data-fashion/luxe-spend.spec.ts

```diff
@@ -36,12 +36,12 @@ describe('with the preset `luxeSpend`', () => {
         "validFrom": undefined,
         "validUntil": undefined,
         "value": {
-          "money": {
-            "centAmount": 3000,
-            "currencyCode": "EUR",
-            "fractionDigits": 2,
-            "type": "centPrecision",
-          },
+          "money": [
+            {
+              "centAmount": 3000,
+              "currencyCode": "EUR",
+            },
+          ],
```

Comment:
More money shape shenanigans, these drafts need to be an array and don't use `CentPrecisionMoney`